.github/workflows/update-prod.yaml

```diff
@@ -95,7 +95,7 @@ jobs:
       run: |
         curl -fL https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/v4.5.5/kustomize_v4.5.5_linux_amd64.tar.gz| tar xz && \
         chmod +x kustomize
-        export TAG=$(curl -H "Accept: application/vnd.github+json" -H "Authorization: token ${{ env.github_token }}" https://api.github.com/repos/brudfyi/flow-voting-tool-argo/releases/latest | jq -r '.tag_name')
+        export TAG=$(curl -H "Accept: application/vnd.github+json" -H "Authorization: token ${{ env.github_token }}" https://api.github.com/repos/DapperCollectives/CAST/releases/latest | jq -r '.tag_name')
```

Comment:
@fersan1985 There is another line below that uses this repo link as well that needs to be changed

Reply:
😅 Please check it now.

Reply:
It should be fine now.